packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -686,10 +683,15 @@ export default {
                     url: this.updateCheckoutUrl,
                     data,
                     timeout: this.checkoutTimeout
+                }).then(headers => {
+                    if (headers) {
+                        this.checkoutAnalyticsService.trackLowValueOrderExperiment(headers);
+                    }
                 });
 
```

Comment:
Can this not be changed to 
```suggestion
               const headers = await this.updateCheckout({
                    url: this.updateCheckoutUrl,
                    data,
                    timeout: this.checkoutTimeout
                })
```